apis/python/node/src/lib.rs

```diff
@@ -84,10 +91,11 @@ impl Node {
     ///                 case "image":
     /// ```
     ///
+    /// Default behaviour is to timeout after 2 seconds.
+    ///
     /// :rtype: dora.PyEvent
-    pub fn __next__(&mut self, py: Python) -> PyResult<Option<PyEvent>> {
-        let event = py.allow_threads(|| self.events.recv(None));
-        Ok(event)
+    pub fn __next__(&mut self, py: Python) -> PyResult<Option<Py<PyDict>>> {
+        self.next(py, Some(2.0))
```

Comment:
This default timeout change does not seem related to this PR. If that's intended, could you open a separate PR for it so that it is more visible?

Reply:
Okay sure!

Reply:
I will leave it to None in that case on this PR